fastify.js

```diff
@@ -594,14 +594,20 @@ function build (options) {
     _fastify.after(function (notHandledErr, done) {
       var path = opts.url || opts.path
       if (path === '/' && prefix.length > 0) {
-        // Ensure that '/prefix' + '/' gets registered as '/prefix'
-        afterRouteAdded('', notHandledErr, done)
+        if (opts.prefixTrailingSlash === undefined) {
+          afterRouteAdded('', notHandledErr, done)
+          afterRouteAdded(path, notHandledErr, done)
+        } else if (!opts.prefixTrailingSlash) {
+          afterRouteAdded('', notHandledErr, done)
+        } else {
+          afterRouteAdded(path, notHandledErr, done)
+        }
```

Comment:
I would prefer we use three different values for `prefixTrailingSlash` using a string. Three-state values using `true`, `false` or `undefined` are hard to reason about.